Makefile

```diff
@@ -176,6 +176,7 @@ release: $(TARGET) meta.xml icon.png
 	$Qmkdir $(RELEASE)/bslug
 	$Qcp -r symbols $(RELEASE)/bslug
 	$Qmkdir $(RELEASE)/bslug/modules
+	$Qcp modules/*/bin/*.mod $(RELEASE)/bslug/modules

```

Comment:
Hmm, I'm not so sure I like this change; it means any brainslug 'release' comes with the modules preinstalled. Whilst this is certainly right for some people, the 'modules' folder was intended more as examples; people don't necessarily want them all by default.

Reply:
I just added that so people who download the CI-built binary will get all the modules as well. But if you don't like that I can remove that from the makefile again and make two CI targets, one that builds Brainslug and one that builds the modules. I'll change that and update this PR when that's done.

Reply:
Done, the Makefile is back to its previous version, and each commit now builds one "brainslug" CI target with the release without modules, and one "modules" CI target with all the module binaries.